### DISCUSSION TITLE
Review image is not "optimal"

### DISCUSSION THREAD

**Discussion by tve** [blakeblackshear/frigate#15318] [Ask A Question]
I have a camera at the entrance of our (long) driveway and the review images are often not great even if the image in the review overview is. For example, in the review overview it shows:
![image](https://github.com/user-attachments/assets/531988ba-e622-4b0c-908d-0b1f1a674dfe)
I can already picture the "car" and "license_plate" bounding boxes... but when I click the little car icon I get the following review image:
![image](https://github.com/user-attachments/assets/f674b7b5-980f-4043-b3a3-c1b5e9c66333)
Any suggestions or explanations?
(I'm using 0.15.0 beta 2 with a pretty fresh frigate+ model. The camera is a dahua 2MP "starlight" and it's basically dark when the photo was taken.)

**Comment by NickM-27** [comment]
Looks like camera settings can be improved considerably, these images are quite blurry. If frigate+ model detects a license plate then that will be used for the object snapshot

**Reply by tve** [reply]
Thanks for the response! I'm not sure how to improve the camera settings. These images were taken at a point where it's basically dark and there is no illuminator around. I have the exposure on auto and backlight is off. This is a color-only (no IR) camera:
![image](https://github.com/user-attachments/assets/20e83da3-d1c4-47f6-8685-4d206b4c2815)
It's possible that my CBR setting is too low and everything gets blurry when essentially the entire image changes due to the sudden headlights, dunno whether that's what's happening. This is a 2MP camera with a 1/8" sensor and I set CBR 4Mb/s.

**Reply by hawkeye217** [reply]
You'll definitely *not* want to use Auto mode for your camera exposure settings in this case.

A user on the ipcamtalk forum has a great post that explains how to tune a camera like this. https://ipcamtalk.com/threads/whats-your-go-to-night-settings-to-start-with.63304/#post-665002

**Reply by tve** [reply]
Thanks for the link. I have that a shot, we'll see... The big annoyance is that now I need to keep adjusting the schedule for the day/night switch. Gonna have to figure out how to automate that, grrr.

**Reply by NickM-27** [reply]
Many cameras have a way to make that automatic based on available light AND an option to set a schedule for the hours on each month so you can account for daylight changes

**Reply by hawkeye217** [reply]
> Thanks for the link. I have that a shot, we'll see... The big annoyance is that now I need to keep adjusting the schedule for the day/night switch. Gonna have to figure out how to automate that, grrr.

All of my Dahua cameras have the ability to switch the day/night settings automatically. I'd assume yours does too. If you are a Home Assistant user, you can also install the Dahua integration and use that to control many aspects of your camera.
